main/src/main/java/cgeo/geocaching/connector/oc/OkapiClient.java

```diff
@@ -809,9 +810,19 @@ private static Geocache parseCache(final ObjectNode response) {
             cache.setLogPasswordRequired(response.get(CACHE_REQ_PASSWORD).asBoolean());
 
             cache.setDetailedUpdatedNow();
+
+            final List<LogEntry> logs = parseLogs((ArrayNode) response.path(CACHE_LATEST_LOGS), cache.getGeocode());
+            final String me = getUserName();
```

Comment:
This won't work in cases where OKAPI username is different from GC username (or GC isn't used at all).

Reply:
pls re-review